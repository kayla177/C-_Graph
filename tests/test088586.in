ENTITY S3KQ5WSN name_S3KQ paper
ENTITY QOE1OHX4 name_QOE1 institution
ENTITY 0H0VMFYY name_0H0V author
RELATIONSHIP S3KQ5WSN presented_at QOE1OHX4 1.4
PRINT S3KQ5WSN
HIGHEST
EXIT
