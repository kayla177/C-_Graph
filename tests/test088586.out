success
success
success
success
QOE1OHX4
S3KQ5WSN QOE1OHX4 1.4
